br/tests/lightning_duplicate_resolution_error/config.toml

```diff
@@ -3,14 +3,14 @@ task-info-schema-name = 'lightning_task_info'
 
 [tikv-importer]
 backend = 'local'
-duplicate-resolution = 'remove'
+duplicate-resolution = 'error'
 add-index-by-sql = false
 
 [checkpoint]
 enable = false
 
 [mydumper]
-batch-size = 1
+batch-size = 2
```

Comment:
should also test the behaviour of `error` that duplicate data are saved in different engines.

Reply:
modified